astropy/visualization/scripts/fits2bitmap.py

```diff
@@ -95,7 +97,11 @@ def fits2bitmap(filename, ext=0, out_fn=None, stretch='linear',
     out_format = os.path.splitext(out_fn)[1][1:]
 
     try:
-        cm.get_cmap(cmap)
+        if minversion(matplotlib, '3.5'):
+            matplotlib.colormaps[cmap]
+        else:
+            from matplotlib import cm
+            cm.get_cmap(cmap)
     except ValueError:
```

Comment:
Drive-by comment: Probably need also to capture `KeyError` for the newer matplotlib versions

Reply:
Good catch. Fixed. Thanks!